shared/src/types/pagination-parameters.type.ts

```diff
@@ -1,6 +1,8 @@
 type PaginationParameters = {
     page?: string;
     limit?: string;
+    totalCount?: number;
+    path?: string;
```

Comment:
why do we need these?

Reply:
These changes are not really necessary. I deleted it.